src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getSingleArticle } from "./api";
+const SingleArticle = () => {
+  const [article, setArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getSingleArticle(article_id).then((response) => {
+      setArticle(response);
+      setIsLoading(false)
+    });
+  }, []);
+if(isLoading) return <p id="status-msg">Content Loading....</p>;
+
+  return (
+    <div className="single-item">
+      <h2>{article.title}</h2>
+      <img src={article.article_img_url} alt={`${article.title}`} />
+      <p>Author: {article.author}</p>
+      <p>{article.body}</p>
+      <p>Comments: {article.comment_count}</p>
+      <p>
+        Created: {article.created_at ? article.created_at.slice(0, 10) : <></>}
+      </p>
+      <p>Topic: {article.topic}</p>
+      <p>Votes: {article.votes}</p>
+    </div>
```

Comment:
Looks good Rob, though in terms of UI you may want to have a look at this article https://www.refactoringui.com/previews/labels-are-a-last-resort
There are often better options than labels for our data, whether it be icons or formatting
You don't need to resubmit but it might be worth using this to improve the UI a bit

Reply:
Thank you for this feedback, I will make sure I implement this in future, I'll concentrate more on this when I focus on the presentation and formatting, I've just updated this slightly here too and will come back to it later.